mlibrary/my_account
# Updating `eslint` and rebuilding the setup.

# Overview
`dependabot` has made its first update, which exposed `eslint-config-standard` not being up to date in over a year. It is not compatible with the latest version of [`eslint`](https://www.npmjs.com/package/eslint). Turns out that initializing `eslint` has changed. This pull request uninstalls all `eslint`-related dependencies and follows the new setup, which added these dependencies:
- `@eslint/js`
- `globals`

These dependencies have been removed:
- `eslint-config-standard`
- `eslint-plugin-import`
- `eslint-plugin-node`
- `eslint-plugin-promise`
- `eslint-webpack-plugin`

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
  - Break the new/updated unit tests to make sure they're working properly.
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Run `npm run eslint` and see if files pass. Make some changes to files that would throw an error and run again.
